lms/resources/_js_config/__init__.py

```diff
@@ -443,7 +443,7 @@ def _get_product_info(self):
             "api": {},
         }
 
-        if product.route.list_group_sets:
+        if self._request.product.settings.groups_enabled:
```

Comment:
Not sure about this conditional.

I reckon this version is better, before it was:

- Return the `group_set_api` if the product is configured for it

While this version is:

- Return the `group_set_api` if groups are enabled.


We could return the endpoint in all cases and the client well decide to call it based on the value of `groupsEnabled` above. :shrug: